napari/utils/events/containers/_nested_list.py

```diff
@@ -449,21 +454,24 @@ def _type_check(self, e) -> _T:
                 )
         return e
 
-    def _iter_indices(self, start=0, stop=None, root=(), deep=False):
+    def _iter_indices(self, start=0, stop=None, root=()):
         """Iter indices from start to stop.
 
         Depth first traversal of the tree
         """
-        if deep:
-            for i in range(start, len(self) if stop is None else stop):
-                if isinstance(self[i], NestableEventedList):
-                    yield from self[i]._iter_indices(  # type: ignore
-                        root=root + (i,), deep=deep
-                    )
-                else:
-                    if root:
-                        yield root + (i,)
-                    else:
-                        yield i
-        else:
-            yield from super()._iter_indices(start, stop)
+        for i, item in enumerate(self[:stop]):
+            yield root + (i,) if root else i
+            if isinstance(item, NestableEventedList):
+                yield from item._iter_indices(root=root + (i,))
```

Comment:
I do not see start usage here. 

`self[start:stop]` or some comment with an explanation?

Reply:
good catch